server/src/test/java/org/elasticsearch/action/admin/indices/close/CloseIndexRequestTests.java

```diff
@@ -11,14 +11,55 @@
 import org.elasticsearch.Version;
 import org.elasticsearch.action.support.ActiveShardCount;
 import org.elasticsearch.action.support.IndicesOptions;
+import org.elasticsearch.common.bytes.BytesArray;
 import org.elasticsearch.common.io.stream.BytesStreamOutput;
 import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.rest.RestRequest;
 import org.elasticsearch.tasks.TaskId;
 import org.elasticsearch.test.ESTestCase;
 import org.elasticsearch.test.VersionUtils;
+import org.elasticsearch.test.rest.FakeRestRequest;
+import org.elasticsearch.xcontent.NamedXContentRegistry;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
 
 public class CloseIndexRequestTests extends ESTestCase {
 
+    public void testIndexPath() throws IOException {
```

Comment:
I think we should have at least some YAML tests in `rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/indices.open/` to test this feature. I think we can probably move all of these tests there.

Reply:
Ok, I added two more YAML tests.